### PR TITLE
Spell checking Hugo docs

### PR DESCRIPTION
**Pull request by gj**

### CODE REVIEW

**Review by gj — COMMENTED** [review]
The submodule & package-lock.json updates aren't from this set of changes.

File: new-docs/content/rust/reference/polar/classes.md [hunk 1]
```diff
@@ -190,11 +190,11 @@ assert!(oso.is_allowed(user, "foo", "bar")?);
...
-| HashMap                   | Dictionary |
-| String, &’static str, str | String     |
+| Rust type                       | Polar type   |
+| ------------------------------- | ------------ |
+| `i32`, `i64`, `usize`           | `Integer`    |
+| `f32`, `f64`                    | `Float`      |
+| `bool`                          | `Boolean`    |
+| `Vec`                           | `List`       |
+| `HashMap`                       | `Dictionary` |
+| `String`, `&’static str`, `str` | `String`     |
```
Comment (plotnick) [inline new-docs/content/rust/reference/polar/classes.md:200]:
```suggestion
| `String`, `&'static str`, `str` | `String`     |
```

File: new-docs/spelling/allowed_words.txt [hunk 2]
```diff
@@ -0,0 +1,154 @@
+# Acronyms
```
Comment (plotnick) [inline new-docs/spelling/allowed_words.txt:1]: I think this should be a simple alphabetical list, and the `make` recipe should enforce this with `sort`. I don't see the categories as adding important information, and they make it harder to use than just sticking a word in the file.

File: new-docs/content/rust/reference/polar/classes.md [hunk 3]
```diff
@@ -97,7 +97,7 @@ using a method, you may iterate over it with [the `in`
 operator](polar-syntax#in-list-membership) or destructure it with [pattern
 matching](polar-syntax#patterns-and-matching).
 
-### HashMaps
+### `HashMap`s
```
Comment (plotnick) [inline new-docs/content/rust/reference/polar/classes.md:100]: "`HashMap`s" typesets poorly. I suggest "Hash Maps" as the section title, and the first line begin with "A Rust `HashMap` is mapped to a Polar dictionary, but require that the key be a `String`:`.

**Review by plotnick — APPROVED** [review]
Nits as noted, but otherwise: thank you for doing this. Tedious work, but a strict improvement now and going forward.

File: new-docs/content/any/project/changelogs/2021-02-10.md [hunk 4]
```diff
@@ -16,7 +16,7 @@ The Go library now supports Windows!
 #### Registering constructors
 
 You can now register a constructor function for a type. This function will be
-called to create new instances when using polar's
+called to create new instances when using Polar's
 [new](https://docs.osohq.com/go/reference/polar-syntax.html#new) operator.
```
[the remark this answers is not in the document]
Comment (samscott89) [reply new-docs/content/any/project/changelogs/2021-02-10.md:20]: Not for this PR. But we should be able to reference this with just `../reference/polar-syntax#new`

File: languages/python/requirements-docs.txt [hunk 5]
```diff
@@ -0,0 +1,11 @@
+Sphinx
+sphinx-copybutton
+sphinx-material
+sphinxcontrib-contentui
+sphinxcontrib-spelling
+git+https://github.com/samscott89/sphinx-notfound-page@master
```
[the remark this answers is not in the document]
Comment (samscott89) [reply languages/python/requirements-docs.txt:6]: Almost certainly not for the Python API docs. But we can revisit when we remove the old docs. Speaking of which... need to add a task for that

File: new-docs/content/any/project/changelogs/2021-02-10.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (gj) [reply new-docs/content/any/project/changelogs/2021-02-10.md:20]: for this PR!

File: new-docs/content/any/_index.md [hunk 6]
```diff
@@ -10,7 +10,7 @@ Oso helps you form a mental model for your
 authorization needs by providing a set of frameworks and templates built on
 top of a
 declarative policy language called Polar. The Oso library provides an
-intuitve API for querying Polar policies, which centralize authorization
+intuitive API for querying Polar policies, which centralize authorization
```
[the remark this answers is not in the document]
Comment (gj) [reply new-docs/content/any/_index.md]: went with simple

File: new-docs/content/rust/reference/polar/classes.md (hunk 3 above)
Comment (gj) [reply new-docs/content/rust/reference/polar/classes.md]: made roughly the suggested change. Don't particularly want to fix now, but there are other spots where we have `backticked code` followed immediately by non-backticked text (usually pluralizing, as was the case here).

File: new-docs/spelling/allowed_words.txt (hunk 2 above)
Comment (gj) [reply new-docs/spelling/allowed_words.txt]: Removed the categories, added a Makefile target to sort the file, and sorted the file. Don't think it matters enough to enforce that the file is sorted in CI.

File: new-docs/spelling/Makefile [hunk 7]
```diff
@@ -7,4 +7,7 @@ spelling-requirements: .make.spelling-requirements.installed
 	pip install -r requirements-spelling.txt
 	touch $@
 
-.PHONY: spelling spelling-requirements
+sort-allowed-words:
```
Comment (plotnick) [inline new-docs/spelling/Makefile:10]: Can we make this a dependency of `spelling-requirements`?

**Comment by plotnick** [comment]
Looks great, thanks.

File: new-docs/spelling/Makefile (hunk 7 above)
Comment (gj) [reply new-docs/spelling/Makefile:10]: c11a808

**Pull request merged**